pkg/subscriptions/insights.jsx

```diff
@@ -0,0 +1,330 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2019 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import cockpit from "cockpit";
+import React from "react";
+import moment from "moment";
+import * as python from "python.js";
+import { show_modal_dialog } from "cockpit-components-dialog.jsx";
+import * as service from "service.js";
+
+import { client } from "./subscriptions-client";
+
+import get_monotonic_start_py from "raw-loader!./get_monotonic_start.py";
+
+let _ = cockpit.gettext;
```

Comment:
Let's make this `const` for cons(t)istency with other sources.

Reply:
Done.